doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -27,6 +27,10 @@ Output types::
 
       filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
       filename: eve.json
+      # Enable GeoIP on source ip and destination ip
+      # Enter full path of GeoLite City database to enrich eve-log with geoip
+      # Disable geoip enrichment by commenting geoip-city-database
+      #geoip-city-database: /usr/share/GeoIP/GeoLite2-City.mmdb
```

Comment:
We already have a "geoip-database" configuration section, is there a reason we need another?  AFAIK, there other can be a city or a country database, depending on what you have access to.

Reply:
Yes, you are right. Perhaps, it is better to merge it with "geoip-database". I was initially thinking that I can save few MBs of memory if this feature is optional due to the implementation of libmaxmindb that use mmap.

If we separate City vs Country we could save memory if enrichment is optional.
For instance these are the size of the databases:
-rw-r--r--.   1 root root  60M Dec 18  2019 GeoLite2-City.mmdb
-rw-r--r--.   1 root root 3.9M Dec 18  2019 GeoLite2-Country.mmdb

Reply:
Would there be a reason why one would want to use city for one, and country for another?

Reply:
Okay, I will revise the code.